src/Javis.jl

```diff
@@ -170,15 +170,17 @@ The current settings of an [`Action`](@ref) which are saved in `action.current_s
 - `opacity::Float64`: the current opacity
 - `mul_opacity::Float64`: the current multiplier for opacity.
     The actual opacity is then: `mul_opacity * opacity`
+- `fontsize::Float64` a
```

Comment:
I think you forgot the rest of the docstring here.

Reply:
Oh yeah I had Real and thought Float64 is fine and then I forgot it. Good catch!